File: addon/java/ArrayListTest.java
import java.util.ArrayList;

public class ArrayListTest {

    public static void main(String[] args) {        
        // 创建ArrayList
        ArrayList<String> list = new ArrayList<String>();

        list.add("1");
        list.add("2");
        list.add("3");
        list.add("4");
				list.add(3, "5");
				
				System.out.println(list.toString());
    }
}

File: addon/java/ArrayListTest.js
const java = require(".")
const ArrayList = java.import('java.util.ArrayList')

let list = new ArrayList()
list.addSync("1")
list.addSync("2")
list.addSync("3")
list.addSync("4")
list.addSync(3, "5")

console.log(list.toStringSync())